src/platform/renesas/KeyValueStoreManagerImpl.cpp

```diff
@@ -207,8 +208,13 @@ CHIP_ERROR KeyValueStoreManagerImpl::_Delete(const char * key)
     cy_rslt_t result = mtb_kvstore_read(&kvstore_obj, key, NULL, NULL);
     if (result == CY_RSLT_SUCCESS)
     {
+        ChipLogProgress(DeviceLayer, "_Delete: Delete existing key %s", key);
```

Comment:
maybe log after it's done and with the result?

Reply:
Move after delete